.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: ruby
 sudo: false
 rvm:
-  - 2.3.4
```

Comment:
Could you please also revert 2.3.x version and add 2.5.0? Then we'll b e sure about support.